Permissioned DEX XLS.md

```diff
@@ -0,0 +1,467 @@
+
+If the `DomainID` is not included in the `OfferCreate` transaction, whenever an offer with a `DomainID` is crossed:
+* The offer outlined in the `OfferCreate` transaction will be checked to see if it is a valid domain offer. If it does, it will fill that offer as per the rules of the `OfferCreate`'s parameters. If it doesn't, the offer will be skipped.
+
+## 7. Transaction: `Payment`
+
+A `Payment` transaction represents a transfer of value from one account to another, and can involve currency conversions and crossing the orderbook. This transaction type already exists on the XRPL, but is being extended as a part of this spec to also support permissioned DEX domains.
+
+### 7.1. Fields
+
+<details>
+<summary>
+
+As a reference, [here](https://xrpl.org/docs/references/protocol/transactions/types/payment) are the existing fields for the `Payment` transaction.
+</summary>
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`Amount`|✔️|`string`|`Amount`|The maximum amount of currency to deliver. For non-XRP amounts, the nested field names MUST be lower-case. If the `tfPartialPayment` flag is set, deliver _up to_ this amount instead.|
+|`DeliverMin`||`string`|`Amount`|The minimum amount of destination currency this transaction should deliver. Only valid if this is a partial payment. For non-XRP amounts, the nested field names are lower-case.|
+|`Destination`|✔️|`string`|`AccountID`|The unique address of the account receiving the payment.|
+|`DestinationTag`||`number`|`UInt32`|An arbitrary tag that identifies the reason for the payment to the destination, or a hosted recipient to pay.|
+|`InvoiceID`||`string`|`Hash256`|An arbitrary 256-bit hash representing a specific reason or identifier for this payment.|
+|`Paths`||`array`|`PathSet`|Array of payment paths to be used for this transaction. Must be omitted for XRP-to-XRP transactions.|
+|`SendMax` ||`string` or `object`|`Amount`|Highest amount of source currency this transaction is allowed to cost, including transfer fees, exchange rates, and [slippage](http://en.wikipedia.org/wiki/Slippage_%28finance%29). Does not include the XRP destroyed as a cost for submitting the transaction. For non-XRP amounts, the nested field names MUST be lower-case. Must be supplied for cross-currency/cross-issue payments. Must be omitted for XRP-to-XRP payments.|.|
+</details>
+
+We propose these additions:
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`DomainID`| |`string`|`Hash256`|The domain that the offer must be a part of.|
+
+#### 7.1.1. `DomainID`
+
+The `DomainID` can only be included if the payment is a cross-currency or partial payment (i.e. if the payment is going to interact with the DEX). It should only be included if the payment is permissioned.
```

Comment:
Why the distinction of `partial payment`? `cross-currency` payment could be a partial payment. Shouldn't direct IOU payments be also permissioned?

Reply:
Direct IOU payments won't touch the DEX, right?

Reply:
It doesn't impact the offers but it changes the trustlines.

Reply:
I don't quite follow why that would need to be permissioned, since all liquidity has already been checked (you set up the trustline to an issuer you trust, and you're sending funds to/from someone you trust).